README.md

```diff
@@ -1,70 +1,183 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
+<div align="center">
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+  <h3><b>CRYPTO TRACKER</b></h3>
 
-## Available Scripts
+</div>
 
```

Comment:
![image](https://github.com/AbdelazizSoliman/cryptonator/assets/93039359/90e05086-1e43-4558-aa40-4b434e184b2a)

- [ ] It would be nice if you rename your pull request title something like React Capstone Complete or Crypto App Finalized etc which is more descriptive 👍

Reply:
Thank you, I updated the title.